scripts/qa_crowbarsetup.sh

```diff
@@ -2055,13 +2055,19 @@ function setup_trusted_cert
     fi
 
     if [ ! -f rootca.pem ] ; then
-        # Generate Root CA key
-        openssl genrsa -aes256 -out rootca.key -passout pass:"crowbar" 4096
-        # Generate Root CA
-        openssl req -x509 -key rootca.key -out rootca.pem \
```

Comment:
adding the "-new" option works for both SP3 and SP4. Don't know why it is not the default for req on SP3.

Reply:
Done.